config/locales/en.yml

```diff
@@ -878,6 +873,9 @@ en:
     label: Course Activity
     last_update: Last statistics update
     next_update: Next estimated update
+    no_update: The Dashboard hasn't imported any data for this program yet
+    enqueued_update: This program has been added to the update queue
+    first_update: Estimated statistics update
```

Comment:
In thinking this through, I realized that this language doesn't quite accurately describe the situation... this time estimate is based on the queue latency, which means it's an estimate of how long until the system *starts* processing the course for a stats update. That's in contrast to the way that it estimates update time after the first update, which is based on the average amount of time between previous updates (so, it's an estimate of when the next update will complete).

For most first updates, this is not a significant difference, but some (if they involve large numbers of edits and/or highly active users) will actually take a long time (minutes to, in extreme cases, hours) to perform the update. Maybe "Estimated start of statistics update" would reflect this better? Or if you can think of any better language, please suggest it.